<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2024, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorTakeRangeForm</class>
 <widget class="QDialog" name="qtractorTakeRangeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>260</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Take Range</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtractor.qrc">:/images/qtractor.svg</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="RangeGroupBox">
     <property name="title">
      <string>Range</string>
     </property>
     <layout class="QGridLayout">
      <item row="6" column="1">
       <widget class="QLabel" name="TakeEndTextLLabel">
        <property name="text">
         <string>En&amp;d:</string>
        </property>
        <property name="buddy">
         <cstring>TakeEndSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QRadioButton" name="SelectionRangeRadioButton">
        <property name="toolTip">
         <string>Selection range</string>
        </property>
        <property name="text">
         <string>&amp;Selection</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QRadioButton" name="EditRangeRadioButton">
        <property name="toolTip">
         <string>Edit range</string>
        </property>
        <property name="text">
         <string>&amp;Edit</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QRadioButton" name="CustomRangeRadioButton">
        <property name="toolTip">
         <string>Custom range</string>
        </property>
        <property name="text">
         <string>&amp;Custom</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="LoopRangeRadioButton">
        <property name="toolTip">
         <string>Loop range</string>
        </property>
        <property name="text">
         <string>&amp;Loop</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="qtractorTimeSpinBox" name="TakeStartSpinBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clip start</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="qtractorTimeSpinBox" name="TakeEndSpinBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clip offset</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="TakeStartTextLLabel">
        <property name="text">
         <string>St&amp;art:</string>
        </property>
        <property name="buddy">
         <cstring>TakeStartSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QRadioButton" name="PunchRangeRadioButton">
        <property name="toolTip">
         <string>Punch range</string>
        </property>
        <property name="text">
         <string>&amp;Punch</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="SelectGroupBox">
     <property name="title">
      <string>Select</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QListWidget" name="CurrentTakeListBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Current take</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="FormatGroupBox">
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QComboBox" name="FormatComboBox">
        <property name="toolTip">
         <string>Time display format</string>
        </property>
        <item>
         <property name="text">
          <string>Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Time</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BBT</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8"/>
 <customwidgets>
  <customwidget>
   <class>qtractorTimeSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qtractorSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SelectionRangeRadioButton</tabstop>
  <tabstop>LoopRangeRadioButton</tabstop>
  <tabstop>PunchRangeRadioButton</tabstop>
  <tabstop>EditRangeRadioButton</tabstop>
  <tabstop>CustomRangeRadioButton</tabstop>
  <tabstop>TakeStartSpinBox</tabstop>
  <tabstop>TakeEndSpinBox</tabstop>
  <tabstop>CurrentTakeListBox</tabstop>
  <tabstop>FormatComboBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc"/>
 </resources>
 <connections/>
</ui>
